gitautosync/cli.py

```diff
@@ -0,0 +1,15 @@
+import argparse
```

Comment:
This shouldn't be a separate file IMO. Just roll everything into one file (even __init__.py?) rather than split them into 3?